server/controllers/githubController.js

```diff
@@ -122,4 +125,4 @@ githubController.getCommits = async (req, res, next) => {
   };
 };
 
-export { githubController };
```

Comment:
I'm not sure why this is showing as changed when there's no change indicated on the line. Same for in githubRouter